README.md

```diff
@@ -123,25 +123,29 @@ await initialize({
 Additionally, this library exposes 23 modules that include the vscode version of some services (with some glue to make it work with monaco):
 
 - **Extensions** (included by default): `vscode/service-override/extensions`
-  - Support for VSCode extensions.
+  - Support for VSCode extensions. A worker configuration can be past to it:
+    - Then, everything runs in one worker, where extensions run in an iframe, with all the implications (can be created by the bundler directly). The worker script is expected to be hosted on a separate domain.
 - **Files** (included by default): `vscode/service-override/files`
-  - It adds the memory filesystem for `file://` files, but also adds the support for lazy loaded extension files, separate memory user files (e.g. config, keybindings), cache, logs, etc.
+  - It adds the memory filesystem for `file://` files, but also adds the support for lazy loaded extension files. It adds separate memory user files (e.g. config, keybindings), cache files and log files.
 - **QuickAccess** (included by default): `vscode/service-override/quickaccess`
-  - Enables the quickaccess menu in the editor
+  - Enables the quickaccess menu in the editor (press F1)
 - **Notifications**: `vscode/service-override/notifications`
-  - This services enables vscode notifications (in the bottom right corner)
+  - This services enables vscode notifications you usually find in the bottom right corner.
 - **Dialogs**: `vscode/service-override/dialogs`
-  - Enable vscode modal dialogs. They allow users to select an action to perform which are exposed to the vscode API. Additionally, this can be used by the language client to ask questions to the user?
+  - Enable vscode modal dialogs. It allows users to select actions to perform and exposed them the vscode API. Additionally, this service can be used by the language client to delegate questions to the user.
```

Comment:
I'm not sure about this sentence:

> It allows users to select actions to perform and exposed them the vscode API.

What is exposed if the possibility to ask something to the user

Reply:
I took a look again what you initially wrote. What about: 
"It allows users to select an action to do. Those actions are exposed to the vscode API..."

I search for something in the vscode docs, but that didn't lead anywhere. Are you ok with this now? ⬇️